charts/stable/guacamole-client/values.yaml

```diff
@@ -206,7 +206,7 @@ initContainers:
         done;
         if [ $DBREADY -eq 1 ];
```

Comment:
THis whole section is useless.
We already have this as an initcontainer.

Reply:
Simply put: The main container AND it's init containers are only started after the database is confirmed to be running.